dpo.obo

```diff
@@ -106,8 +106,8 @@ def: "A phenotype that is an expansion of the developing embryonic nervous syste
 comment: Typically this is caused by and expansion of the population of neuroblasts at the expense of prospective epidermis, both of which originate in the ventral ectoderm.
 subset: fbcvsubset_mgiribbons
 is_a: FBcv:0001347 ! phenotype
-relationship: dc-contributor http://orcid.org/0000-0002-7073-9172
-property_value: dcterms-date 2011-09-13T02:42:10Z xsd:dateTime
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-7073-9172
```

Comment:
This seems like a pretty random diff that I would probably undo

Reply:
Ok, up to you! I have no problems, its just a large diff if your goal is to understand the consequence of understanding base merging.

Reply:
Yeah, I could have done first a mock release _without_ base merging but with the new OWL API, and then a second mock release with base merging so that I could see the effects of base merging only, but that seemed a bit too much.

(And I’ve spent so much time looking at such diffs when I was working on the serialisation stuff for the Great OWL API Migration that by now my brain is automatically ignoring those parts in a diff. :D )